projects/igniteui-angular/src/lib/grids/column-chooser-item-base.ts

```diff
@@ -9,8 +9,7 @@ export interface IValueChangedEventArgs extends IBaseEventArgs {
 /**
  *@hidden
  */
-@Directive()
```

Comment:
Unless I missed some change, this shouldn't be valid in Angular 9 since the class does have bindings (`@Input` decorated value prop) so it should be either a directive or component.